src/styles/index.css

```diff
@@ -39,10 +39,11 @@
 body {
   font-family: 'PilcrowRounded';
   font-size: 16px;
+  color: #2e2e2e;
 
-  @apply bg-gray-50
+  @apply bg-gray-50;
 }
 
-.multiSelectContainer li:hover{
+.multiSelectContainer li:hover {
```

Comment:
can't we move this class right to the needed element and use it there via tailwind?